pybloqs/htmlconv/wkhtmltox.py

```diff
@@ -0,0 +1,116 @@
+        cmd += ["--zoom", str(pdf_zoom)]
+        if pdf_auto_shrink:
+            cmd.append("--enable-smart-shrinking")
+        else:
+            cmd.append("--disable-smart-shrinking")
+
+        # Specific wkhtmltox
+        cmd += ["--no-stop-slow-scripts", "--debug-javascript"]
+        cmd += ["--javascript-delay", str(kwargs.get('javascript_delay', 200))]
+        [cmd.extend([k, v]) for k, v in kwargs.items()]
+
+        # Render without header and footer as those are handles explicitly below
+        content = block.render_html(static_output=True)
+        name = block._id[:ID_PRECISION] + ".html"
+        tempdir = tempfile.gettempdir()
+        html_filename = os.path.join(tempdir, name)
+        with open(html_filename, "w") as f:
+            f.write(content)
+
+        if header_block is not None:
+            header_file = header_block._id[:ID_PRECISION] + ".html"
+            header_filename = header_block.publish(os.path.join(tempdir, header_file))
+            cmd += ['--header-html', header_filename]
+            cmd += ['--header-spacing', str(header_spacing)]
+
+        if footer_block is not None:
+            footer_file = footer_block._id[:ID_PRECISION] + ".html"
+            footer_filename = footer_block.publish(os.path.join(tempdir, footer_file))
+            cmd += ['--footer-html', footer_filename]
+            cmd += ['--footer-spacing', str(footer_spacing)]
+
+        cmd.extend([html_filename, output_file])
+
+        output, errors = self.run_command(cmd)
+        return output, errors
+
+
+class WkhtmltoimageConverter(HTMLConverter):
+
+    def htmlconv(self, block, output_file, pdf_zoom=1, **kwargs):
+        """
+        Use the wkhtmltoimage tool to convert the supplied HTML content to an image file.
+
+        :param block: The block to render
+        :param output_file: File name for converter output.
+        :param pdf_zoom: The zooming to apply when rendering the page.
+        :param kwargs: Additional parameters. Passed to wkhtmltoimage.
+        """
+
+        content = block.render_html(static_output=True)
+        name = block._id[:ID_PRECISION] + ".html"
+        tempdir = tempfile.gettempdir()
+        html_filename = os.path.join(tempdir, name)
+        with open(html_filename, "w") as f:
+            f.write(content)
+
+        cmd = []
+        cmd.append(self.get_executable('wkhtmltoimage'))
+        cmd += ["--format", os.path.splitext(output_file)[1][1:]]
+        cmd += ["--zoom", str(pdf_zoom)]
+        # Set width to 0 to make it fit the content exactly, without any extra margin.
+        cmd += ["--width", "0"]
+
+        # Specific wkhtmltox
+        cmd += ["--no-stop-slow-scripts", "--debug-javascript"]
+        cmd += ["--javascript-delay", str(kwargs.get('javascript_delay', 200))]
+
+        # Remove default parameters that are not relevant for image output and pass the rest to wkhtmltoimage
+        extra_params = kwargs.copy()
+        del extra_params['header_block']
```

Comment:
Maybe pull these out into a blacklist constant and use
[cmd.extend([k, v]) for k, v in kwargs.items() if k not in IMAGE_KWARG_BLACKLIST]